docs/manta/Technology.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 1
+title: Manta’s Architectural Design Choices
+id: Technology
+---
+
+With the Ethereum 1.0 main net launch in 2015, blockchain entered into a new, Turing-complete chapter. Fast-forward to now, and we are experiencing a tremendous level of adoption for on-chain applications, especially regarding decentralized finance (DeFi). While existing decentralized projects and applications prove the need and demonstrate a market for DeFi, we believe that adoption is only beginning.
+
+To take DeFi to the next level, we need to address the existing issues: the lack of privacy, speed, and interoperability. Manta intends to expand DeFi into a larger market with better conveniences and privacy provisions by introducing a top-tier product that addresses these problems.
+
+![Polkadot](/img/polkadot.jpg)
+
+## Choosing Polkadot
+
+To address those issues in the most efficient manner possible, we are excited to build our project as a Polkadot parachain. To enable fast and privacy-preserving transactions, we believe the underlying architecture must fulfill three characteristics that are available in Polkadot:
+
+1. First, we need a fast runtime. We leverage zk-SNARK to guarantee privacy-preserving transactions on-chain. Although the verification process in the newer zk-SNARK schemes such as Groth16 is relatively quick and cheap, it can still be computationally intensive and taxing on other blockchains. Substrate uses a WASM-based runtime that supports compiling native Rust code to WASM. This makes it more efficient for our purposes than EVMs.
+
+![Choosing Polkadot](/img/choosing_polkadot.png)
+
+2. Second, we require a fast consensus to provide the best user experience possible with our privacy-preserving decentralized exchange (DEX) and transacting. We want to achieve trustlessness through decentralization; as a result, we are building a layer-1 (L1) solution, which will reap the benefits of a fast consensus protocol. Polkadot’s GRANDPA consensus is one of the quickest consensus protocols available.
```

Comment:
Shouldn't we mention BABE when mentioning GRANDPA.

Reply:
Can you give me more context?

Reply:
The consensus has two parts:
1. Block production (BABE) https://medium.com/polkadot-network/polkadot-consensus-part-3-babe-dcc2e0dd8878
2. Finalization (GRANDPA) https://medium.com/polkadot-network/polkadot-consensus-part-2-grandpa-fb1963ef6c70

The fact that that the two processes are isolated contributes to the fast-ness of the protocol that is mentioned here.

We're going to be using BABE+GRANDPA in the future, right ? @stechu